src/test/regress/expected/follower_single_node.out

```diff
@@ -226,15 +232,10 @@ SELECT * FROM ref, local WHERE a = c ORDER BY a;
  7 | 8 | 7 | 8
 (2 rows)
 
--- Check repartion joins are support
+SET citus.enable_repartition_joins TO ON;	
 SELECT * FROM test t1, test t2 WHERE t1.x = t2.y ORDER BY t1.x;
-ERROR:  floating-point exception
-DETAIL:  An invalid floating-point operation was signaled. This probably means an out-of-range result or an invalid operation, such as division by zero.
-SET citus.enable_repartition_joins TO ON;
-SELECT * FROM test t1, test t2 WHERE t1.x = t2.y ORDER BY t1.x;
-ERROR:  floating-point exception
-DETAIL:  An invalid floating-point operation was signaled. This probably means an out-of-range result or an invalid operation, such as division by zero.
-RESET citus.enable_repartition_joins;
+ERROR:  writing to worker nodes is not currently allowed
+DETAIL:  the database is in recovery mode
```

Comment:
Can we add tests for:
- single hash repartition joins
- Changing `citus.task_assignment_policy`